### ISSUE TITLE
Improve SEO to appear in first tem results of Google search for keywords Jeevan Rakht

### ISSUE THREAD

**Issue by strongSoda** [UdacityFrontEndScholarship/jeevan-rakht#211]
Improve the SEO tags and make Jeevan Rakht more visible  in Google search results. Optimise code by minifying , images , etc .

**Comment by vattytrivedi** [comment]
We can make use of the OG meta tags.
If you wish to implement I will drop the code for it.

**Comment by strongSoda** [comment]
Yes go ahead and also try to come up with  other ways to increase visibility.

**Comment by pm17** [comment]
Bookmarking helps a lot to get better ranks, so we can even have social media pages for the web app, if mods approve and it is with in the scope of project for time being.

**Comment by vattytrivedi** [comment]
I am making this PR to improve the search results and also making it more shareable - getting more contents like favicon and info when we try to share the link on WhatsApp/Slack or any other platform.
The search tags used also increases the search result in a google search.

Add meta tags: OG and itemprop #225

**Comment by vattytrivedi** [comment]
@strongSoda Can you have a look at the PR?

**Comment by gauravsinghaec** [comment]
@vattytrivedi please refrain from raising PR before getting approval on issues.
I would request you to share the screenshots or link to see before and after effect. Hope this makes sense.

**Comment by vattytrivedi** [comment]
@gauravsinghaec 
For this to work you have to accept the PR as i have set the tags for this repository and project. 
So if you accept it i will be able to show you the results.

**Comment by gauravsinghaec** [comment]
@vattytrivedi why is it dependent on our acceptance, sorry but please deploy on heroku and share your heroku test link. Just let me know that if it is not possible to test on your heroku link with the corresponding tag of your repo.

**Comment by vattytrivedi** [comment]
Here are the tags that i have used.
  
+    <meta itemprop="name" content="JeevanRakht">
+    <meta itemprop="description"
+            content="JeevanRakht is a Virtual Blood Bank web app">
+    <meta itemprop="image" content="https://jeevanrakht.herokuapp.com/assets/images/favicon.png">
+
+    
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="JeevanRakht">
+    <meta property="og:image" content="https://jeevanrakht.herokuapp.com/assets/images/favicon.png">
+    <meta property="og:description"
+            content="JeevanRakht is a Virtual Blood Bank web app">
+    <meta property="og:site_name" content="JeevanRakht">
+    <meta property="og:url" content="https://jeevanrakht.herokuapp.com/"/>
+

Now the thing is the original repo is jeevn-rakht for which i have made according changes. Now if i wish to do for my repo then i have to change the tags that is the first thing. And secondly the repo code is same so when you will search for same link of this original repo then that will conflict with the results of my version. 
So i wish if you could trust on this one and merge the PR, it will be helpful.

**Comment by gauravsinghaec** [comment]
Hey @vattytrivedi only change you need to do to show us is that just updates the below URL with your heroku URL and then you can share the screenshot of the improvements you have done
https://**jeevanrakht**.herokuapp.com

**Comment by vattytrivedi** [comment]
See i have not hosted the project on heroku i.e. the first thing. 
Secondly the tags results depend on the number of visits and making changes in my repo and hosting it no one is going to visit it. And members use this repo link daily so it will give much faster results.
So, now it is up to you. I will not be able to give you the screenshot of the result as there is any 2 days left. I can assure you if you accept the PR i will be able to show you result in 1 day.

**Comment by viditkothari** [comment]
@vattytrivedi, May I ask for confirmation:
If there are two files with 15 additions: 1) .PNG file for the favicon and 2) meta tags which help in SEO, so I'm guessing the changes aren't viewable.

Secondly, what is it about the two repos. that you are referring to? Is there any conflicting issue that I need to be aware of?

When I feel these additions are good to be included, and not conflicting I'll process the merge request :)

**Comment by vattytrivedi** [comment]
Yes there are only 2 changes.
Have a look at the below link
http://ogp.me
And the reference to the two repo is not about conflicting. The tags work on keywords if both repos have same tag then they will display different  results instead of only one website.
And i don't wish to debate more on this topic.
I will wait for some time after is comment and will remove the request if you all don't feel it useful.
Thanks and sorry.

**Comment by viditkothari** [comment]
@vattytrivedi,
I'm unable to understand which two repos are you referring to when you say *both repo* : `The tags work on keywords if both repos have same tag then they will display different results instead of only one website`

@strongsoda created this issue to improve Google ranking but your code seems to deal with Facebook's Open graph, about which I'm not sure if it will yield any effective result in short duration.

**Comment by gauravsinghaec** [comment]
@vattytrivedi Can we see the results of improvement now as your changes are there in original site for few days now?

**Comment by vattytrivedi** [comment]
Following is the result if you share the link of the website.

![photogrid_1526535889534](https://user-images.githubusercontent.com/14287645/40158727-b0d6fe90-59c3-11e8-8b89-6b28e8295dfe.jpg)

And you make a  google search the result will be 2nd or 3rd for the link.

**Comment by gauravsinghaec** [comment]
Hey that's great and just see below stats 
![image](https://user-images.githubusercontent.com/15084301/40158974-e1717a7a-59c4-11e8-868f-9179df8a0693.png)
Why is the GitHub link coming up first rather the site link
**https://jeevanrakht.herokuapp.com/**?

**Comment by gauravsinghaec** [comment]
Google search on jeevanrakht.herokuapp.com is quite good. But @vattytrivedi also is it not possible to improve the Google search on keyword **Jeevan Rakht** as mentioned in issue title. Check below:
![image](https://user-images.githubusercontent.com/15084301/40159280-2d5c00f8-59c6-11e8-9ded-3742bce9b889.png)

**Comment by vattytrivedi** [comment]
https://www.google.com/webmasters/tools/home?hl=en

fill the details of the website in this link and add the generated tag.
